src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -110,6 +112,11 @@ public virtual void Reset()
             _settingsCache.Clear();
         }
 
+        public virtual bool GetBooleanSetting(string settingKey)
+        {
+            return GetSetting(settingKey) == "1";
```

Comment:
Won't this only work if it's set to "1"? We need to check whether this App Setting is set at all.